configuration/peloton.conf

```diff
@@ -0,0 +1,119 @@
+# -----------------------------
```

Comment:
Where did you load this file for the server? I just didn't find the code... Did you specify it somewhere in the CMake file?